### PR TITLE
Update AppendixA.md

### PR DESCRIPTION
**Pull request by michaelhortongsa**
Updated alignment of baseline tests (1-24) to the test to Section508/WCAG requirements.

### CODE REVIEW

**Comment by kengdoj** [comment]
Issue #250

File: docs/AppendixA.md [hunk 1]
```diff
@@ -6,31 +6,30 @@ Baseline to Section 508/WCAG Requirement
...
+| 17. Synchronized Media        | 1.2.2 Captions (Prerecorded); 1.2.3 Audio Description or Media Alternative (Prerecorded); 1.2.4 Captions (Live); 1.2.5 Audio Description (Prerecorded); Section 508 503.4 User Controls for Captions and Audio Description; Section 508  503.4.1 Caption Controls; Section 508 503.4.2 Audio Description Controls                      |
+| 18. Stylesheet Non-dependence | 1.1.1. Non-Text Content; 1.3.1 Info and Relationships; 1.3.2 Meaningful Sequence                                           |
 | 19. Frames and iFrames        | 4.1.2 Name, Role, Value                                            |
-| 20. Alternate Versions        | WCAG Conformance Requirement 1. Conformance Level                  |
-| 21. Timed Events              | 1.4.2 Audio Control; 2.2.1 Timing Adjustable; 2.2.2 Pause, Stop, Hide; 4.1.2 Name, Role, Value                                             |
-| 22. Resize Text               | 1.4.4 Resize text                                                  |
+| 20. Alternate Versions        | WCAG Conformance Alternate Version                                 |
```
Comment (cliffbert) [inline docs/AppendixA.md:28]: Why this change, since elsewhere you are using a different format, e.g. "Conformance Requirement 5: Non-Interference"?

File: docs/AppendixA.md [hunk 2]
```diff
@@ -6,31 +6,30 @@ Baseline to Section 508/WCAG Requirement
...
+| 9. Flashing                   | 2.3.1 Three Flashes or Below Threshold; Conformance Requirement 5: Non-Interference                             |
+| 10. Forms                     | 1.3.1 Info and Relationships; 2.4.6 Headings and Labels; 3.2.2 On Input; 3.3.1 Error Identification; 3.3.2 Labels or Instructions; 3.3.3 Error Suggestion; 3.3.4 Error Prevention (Legal, Financial, Data); 4.1.2 Name, Role, Value                                             |
 | 11. Page Titles               | 2.4.2 Page Titled                                                  |
 | 12. Tables                    | 1.3.1 Info and Relationships                                       |
 | 13. Content Structure         | 1.3.1 Info and Relationships; 2.4.6 Headings and Labels                                           |
-| 14. Links                     | 2.4.4 Link Purpose (In Context); 4.1.2 Name, Role, Value                                             |
+| 14. Links                     | 2.4.4 Link Purpose (In Context)                                             |
```
Comment (cliffbert) [inline docs/AppendixA.md:22]: Could you remind me why 4.1.2 is removed, since it mentions links  (generated by scripts)? Are we relying on 2.4.4 to cover this?

Comment (kengdoj) [reply docs/AppendixA.md:22]: Cliff - you're right. It shouldn't have been removed from the top of test 14. SC 4.1.2 is referenced in instruction 1. I'll make that edit. Sorry, Michael, please add 4.1.2 back in.

**Comment by kengdoj** [comment]
approving this, with additional changes to come

**Review by kengdoj — APPROVED** [review]
